akka-http2-support/src/main/scala/akka/http/scaladsl/Http2.scala

```diff
@@ -60,36 +60,41 @@ class Http2Ext(private val config: Config)(implicit val system: ActorSystem) ext
 
     val effectivePort = if (port >= 0) port else 443
 
-    val serverLayer: BidiFlow[HttpResponse, ByteString, ByteString, HttpRequest, NotUsed] = {
-      val wrappedContext = WrappedSslContextSPI.wrapContext(httpsContext)
-      val tls = http.sslTlsStage(wrappedContext, TLSRole.server)
+    def http2Layer(): BidiFlow[HttpResponse, SslTlsOutbound, SslTlsInbound, HttpRequest, NotUsed] =
```

Comment:
Actually, this didn't trigger the discussion (the original trigger of the discussion was about being able to tag a graph with metadata like the localAdress/remoteAddress to use that in log message) but I agree that it would be useful here as well.